src/sentry/monkey/pickle.py

```diff
@@ -96,7 +96,7 @@ def patch_pickle_loaders():
     # XXX(epurkhiser): BIG IMPORTANT NOTE! When changing this, we will have to
     # make some updates to our data pipeline, which currently uses 'pickle.js'
     # to depickle some data using javascript.
-    pickle.DEFAULT_PROTOCOL = 2
+    pickle.DEFAULT_PROTOCOL = 5
```

Comment:
Just did this to make it easier to test what I needed, but I do have this up https://github.com/getsentry/sentry/pull/30390

Reply:
So if I get this right, the plan is to wait for #30390  before deploying this one ?

Reply:
ping